Teacher Workout/Features/Onboarding/View/OnboardingPageViewController.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+
+class OnboardingPageViewController: UIPageViewController, OnboardingViewControllerDelegate {
+    let pages = OnboardingPage.mockedData
```

Comment:
Use private for variables that are not accessed outside the scope of this class.